gradle/libs.versions.toml

```
@@ -14,6 +14,7 @@ jmock = { group = "org.jmock", name = "jmock", version.ref = "jmock" }
 junit-jupiter-api = { group = "org.junit.jupiter", name = "junit-jupiter-api", version.ref = "junit" }
 junit-jupiter-engine = { group = "org.junit.jupiter", name = "junit-jupiter-engine", version.ref = "junit" }
 junit-jupiter-params = { group = "org.junit.jupiter", name = "junit-jupiter-params", version.ref = "junit" }
+junit-platform = { group = "org.junit.platform", name = "junit-platform-launcher" }
```

Don't we want to have the version here too?

```suggestion
junit-platform = { group = "org.junit.platform", name = "junit-platform-launcher", version.ref = "junit" }
```

---

I don't think so. See: https://junit.org/junit5/docs/current/user-guide/#running-tests-build-gradle-bom

---

>Since all JUnit artifacts declare a [platform](https://docs.gradle.org/current/userguide/platforms.html) dependency on the BOM, you usually don’t need to declare an explicit dependency on it yourself. Instead, it’s sufficient to declare one regular dependency that includes a version number. Gradle will then pull in the BOM automatically so you can omit the version for all other JUnit 5 artifacts.

---

Then again, it does not hurt and in our case with our version catalogue it does make things more consistent so I will add it anyway. :-)

---

I added a comment now about this